libreoffice-core/desktop/source/lib/init.cxx

```diff
@@ -1118,6 +1118,7 @@ static void doc_setGraphicSelection (LibreOfficeKitDocument* pThis,
                                   int nY);
 static void doc_resetSelection (LibreOfficeKitDocument* pThis);
 static char* doc_getCommandValues(LibreOfficeKitDocument* pThis, const char* pCommand);
+static char* doc_gotoOutline(LibreOfficeKitDocument* pThis, int idx);
```

Comment:
As I said, I'm not a fan of their whole fragile API structure and I think it would be better placed in postUnoCommand for one-off's like this, but if it works, it works.

Reply:
I agree, it would be better to be in uno but I think for this it's a true one off since we need the data returned from it and this was a lot easier than hooking up a uno event for this one instance.